README.md

```diff
@@ -27,6 +27,17 @@ You can also run any `haskellings` command from the project directory using `sta
 >> stack exec -- haskellings run Types1
 ```
 
+### Using Nix
```

Comment:
Let's add in this section that the usage is still a bit experimental.

Reply:
Just revised the passage! Let me know how it looks now.

Reply:
Looks good!

Reply:
Looks good!